adagrams/game.py

```diff
@@ -1,13 +1,95 @@
 from random import randint
 
 def draw_letters():
-    pass
+    # Letter distribution as per the provided table
+    LETTER_DISTRIBUTION = {
+        'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1, 'K': 1,
+        'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2,
+        'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    }
+
+    # Create a pool of letters based on their frequency
+    LETTER_POOL = []
```

Comment:
In a similar vein, you indicate that `LETTER_POOL` is a constant by writing it in all capital letters. You then go on to assign it an empty list which means you are likely going to change it later. Even though we technically can modify a constant without Python interfering, it is against best practices to do so. It is pretty unheard of that we will have a constant that is an empty data structure of any sort (list, dictionary, tuple, etc.). 

```suggestion
    letter_pool = []
```